pkg/hooks/hooks.go

```diff
@@ -144,5 +147,9 @@ func (h *HookDir) runHooks(dirpath string, hType HookType, pod Pod, podManifest
 
 func (h *HookDir) RunHookType(hookType HookType, pod Pod, manifest *pods.Manifest) error {
 	dirpath := path.Join(h.dirpath, hookType.String())
-	return h.runHooks(dirpath, hookType, pod, manifest)
+	if err := h.runHooks(dirpath, hookType, pod, manifest); err != nil {
```

Comment:
Requested cleanup: distinguish `h.dirpath` and `dirpath` more clearly - not obvious that one is global